### DISCUSSION TITLE
Get/Set-AuthenticodeSignature in PowerShell 7 for Linux

### DISCUSSION THREAD

**Discussion by bitofrumncoke** [PowerShell/PowerShell#15148] [Q&A]
TLDR: Any idea on when PowerShell 7.x will have Get-AuthenticodeSignature and Set-AuthenticodeSignature on Linux? 

I have a centralized script that I host on a tightly controlled server of which thousands of Windows clients pull down and execute to gather system information and send back to Splunk via HEC. However, all the clients check the scripts signature to ensure it's valid signed code prior to executing. Would love to do this on Linux as well so it would be terrific to have these cmdlets on the Linux side as well. Found tools like chktrust (early 2000's/not in RHEL repo's) and osslsigncode (again, nothing in RHEL repo of course) but I strongly prefer to "live off the land" with tools on the OS (or ones in a repo I can maintain) vs. having to install multiple other tools and have to maintain them separately (PowerShell aside). 

Mentioned here but no updates:
https://github.com/PowerShell/PowerShell/blob/b25b8df7cae323c376fe4770882ebbfd805fdbf4/docs/KNOWNISSUES.md#command-availability
